README.md

```diff
@@ -87,6 +87,10 @@ An example configuration can be found [here](circuitbreaker-example.yaml)
 * Execute `circuitbreaker` with the correct command line flags to connect to
   `lnd`. See `circuitbreaker --help` for details.
 
+### Installation guide
+
+* [Circuit Breaker on the RaspiBolt](https://raspibolt.org/bonus/lightning/circuit-breaker.html): Manually install Circuit Breaker on any Debian-based OS.
```

Comment:
That is a nice guide indeed. Link is broken now though and should be https://raspibolt.org/guide/bonus/lightning/circuit-breaker.html I think.

Btw, I am removing hold fee simulation from circuitbreaker as it isn't core functionality. The guide is describing that part too.

Reply:
> That is a nice guide indeed.

Thanks ❤️ 
Feel free to suggest improvements if you think of any.

> Link is broken now though and should be https://raspibolt.org/guide/bonus/lightning/circuit-breaker.html I think.

Fixed

> Btw, I am removing hold fee simulation from circuitbreaker as it isn't core functionality. The guide is describing that part too.

Thanks, fixed with [#1131](https://github.com/raspibolt/raspibolt/pull/1131).